src/Build/Evaluation/LazyItemEvaluator.IncludeOperation.cs

```diff
@@ -97,6 +100,7 @@ protected override ImmutableList<I> SelectItems(ImmutableList<ItemData>.Builder
                         }
 
                         string[] includeSplitFilesEscaped;
+                        MSBuildEventSource.Log.SelectItemsStart();
```

Comment:
Include `_rootDirectory`, `glob` and if not too difficult within the `TraceEvent` API `excludePatternsForGlobs`. That's critical info if this operation turns out to take a long time.